bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/interactive/ConvertTask.java

```diff
@@ -0,0 +1,208 @@
+        List<String> parameterList = new ArrayList<>(Arrays.stream(parameters).toList());
+        parameterList.remove(0);
+
+        ConversionTable conversionTable = new ConversionTable();
+        Converter converter = conversionTable.getConverter(ConversionKey.of(origin.get(), destination.get()));
+        ConverterModel converterModel = getConverterModel(origin.get(), parameterList);
+
+        PersistableConverterModel persistableConverterModel = converter.convert(converterModel);
+
+        if (parameterList.size() != 2) {
+            System.err.println("Need two additional parameter for saving the converted model, but got: " + parameterList.size());
+            System.exit(-1);
+        }
+        String filePath = parameterList.remove(0);
+        String fileName = parameterList.remove(0);
+        persistableConverterModel.save(filePath, fileName);
+    }
+
+    private static void handleInteractive() {
+        Scanner scanner = new Scanner(System.in);
+        ModelType origin = getOrigin(scanner);
+        ModelType goal = getGoal(scanner, origin);
+        runConversion(origin, goal, scanner);
+    }
+
+    private static ModelType getOrigin(Scanner scanner) {
+        System.out.println("Please enter the desired model: ");
+        ConversionTable conversionTable = new ConversionTable();
+        List<ModelType> origins = conversionTable.getPossibleOrigins()
+                .stream()
+                .toList();
+        return getModelTypeFromSelection(scanner, origins);
+    }
+
+    private static ModelType getGoal(Scanner scanner, ModelType origin) {
+        System.out.println("Please enter the desired model: ");
+        ConversionTable conversionTable = new ConversionTable();
+        List<ModelType> goals = conversionTable.getPossibleDestinations(origin)
+                .stream()
+                .toList();
+        return getModelTypeFromSelection(scanner, goals);
+    }
+
+    private static ModelType getModelTypeFromSelection(Scanner scanner, List<ModelType> modelTypes) {
+        for (int i = 0; i < modelTypes.size(); i++) {
+            System.out.println("[" + (i + 1) + "] " + modelTypes.get(i)
+                    .getName());
+        }
+        try {
+            int index = Integer.parseInt(scanner.nextLine()) - 1;
+            return modelTypes.get(index);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(e);
+        }
+    }
+
+    private static void runConversion(ModelType origin, ModelType goal, Scanner scanner) {
+        ConversionTable conversionTable = new ConversionTable();
+        Converter converter = conversionTable.getConverter(ConversionKey.of(origin, goal));
+
+        ConverterModel input = getConverterModelInteractive(origin, scanner);
+        PersistableConverterModel output = converter.convert(input);
+
+        System.out.println("Please enter a path to a folder where the files should be saved. Leave empty for current working directory");
```

Comment:
All methods that accept file paths (both input and output) should accept paths in quotes, i.e., `"path/to/file"` instead of only accepting `path/to/file`. This simplifies the workflow on windows where users could just use File -> Copy Path.

Reply:
Done in 221cc1f17f57